### PR TITLE
Pan and zoom, riiif upgrade, deploy scripts

### PR DESCRIPTION
**Pull request by val99erie**

### CODE REVIEW

**Comment by dunn** [comment]
How does this PR relate to https://github.com/ucsblibrary/alexandria/pull/104?

**Comment by val99erie** [comment]
@dunn Ah, I think @bess and I were both trying to put in the bulk PR.  I'll close #104.  I think this PR covers everything that would be in that PR.

File: app/views/catalog/_media_image.html.erb [hunk 1]
```diff
@@ -18,7 +18,12 @@
       <% if document.file_sets.find { |fs| fs.id == fs_id }.pdf? %>
         <%= link_to(image_tag(img, style: 'width: 100%'), download_url(fs_id, only_path: true)) %>
       <% else %>
-        <%= image_tag(img, style: 'width: 100%') %>
+        <div class="row zoom">
```
Comment (dunn) [inline app/views/catalog/_media_image.html.erb]: I asked on Bess' PR:
> What does this look like when there are multiple images for an object, and the JavaScript carousel code takes effect?

**Comment by val99erie** [comment]
@dunn I talked to @bess after you had your meeting.  It sounds like we are going to re-submit these commits in separate PRs.  So, I'll close this PR without merging.

**Pull request closed without merging**